### DISCUSSION TITLE
Commissions

### DISCUSSION THREAD

**Discussion by sleepless-ted** [kernc/backtesting.py#551] [Q&A] [answered]
How can I access to the commission value set in the Backtest() function inside my strategy?

I've tried something like ```self.commission``` or ```self._commission``` but it doesn't work

**Comment by kernc** [comment, marked as answer]
It's `self._broker._commission`, but this is not public API so use it sparingly. :smiley:

**Reply by sleepless-ted** [reply]
Thanks a lot! I love your work

**Comment by samueltg92** [comment]
Hello kernc, how can I access to the entry_price value?

**Reply by kernc** [reply]
Have you seen [Quick Start tutorial, _Trade data_ section](https://kernc.github.io/backtesting.py/doc/examples/Quick%20Start%20User%20Guide.html#Trade-data)?

**Reply by samueltg92** [reply]
I mean, access to entry_price value so I been able to use that entry price to calculate a StopLoss, for example... BTW I fckn love this library!
